Headlines/src/Controllers/NewsPreviewViewController.swift

```diff
@@ -15,39 +15,39 @@ class NewsPreviewViewController: UIViewController {
     @IBOutlet weak var imageView: UIImageView!
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var subtitleLabel: UILabel!
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
         imageView.sd_setImage(with: news?.imageUrl)
         titleLabel.text = news?.title
         subtitleLabel.text = news?.source
     }
-    
+
     override var previewActionItems: [UIPreviewActionItem] {
-        
+
         let shareAction = UIPreviewAction(title: "Compartir Noticia", style: .default, handler: { (_, _) -> Void in
             guard let n = self.news else {
                 return
             }
-            
+
             UIPasteboard.general.string = ShareCanillitapActivity.canillitappURL(fromNews: n)
         })
-        
+
         var actions = [shareAction]
-        
+
         if newsViewController != nil {
             let reactionsAction = UIPreviewAction(title: "Agregar Reacción", style: .default, handler: { (_, _) -> Void in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/adb6600e04f8d78e85d3ed821c2844be0fbd4321/Headlines/src/Controllers/NewsPreviewViewController.swift#L40"></span>Line should be 120 characters or less: currently 122 characters<br />
<code>line_length</code> <code>NewsPreviewViewController.swift:40</code></td>
    </tr>
  </tbody>
</table>